public_html/assets/js/nf-core.js

```diff
@@ -188,25 +188,10 @@ $(function () {
     // Make the stats tables sortable
     $('.pipeline-stats-table').tablesorter();
 
-    // version number injection in the URL in the docs
-    if ($('#version_select').length >0){
-        if (document.location.href.match(/\/\d+\.\d+|\/dev/)){
-            //update selected option in select dropdown if version was found in the url
-            var pipeline = $('#version_select').data('pipeline');
-            var url = document.location.href.split(pipeline);
-            var selected_version = url[1].match(/^\/(\d+\.\d+\.\d+)|^\/(\d+\.\d+)|^\/(dev)/)[0]; // extract version from url
-            selected_version = selected_version.replace("/","");
-            $('#version_select option[value="' + selected_version + '"]').attr("selected", true); // update selected option in select input
-        }
-        // Add selected pipeline version to URL
-        $('#version_select').on('change',function(){
-            var selected = $('option:selected',this).text();
-            var pipeline = $('#version_select').data('pipeline');
-            var url = document.location.href.split(pipeline);
-            url[1] = url[1].replace(/^\/\d+\.\d+\.\d+|^\/\d+\.\d+|^\/dev/,"/"); // remove previously added version in url
-            document.location.href = url[0] + pipeline + "/" + selected + url[1]; // add selected version into url after pipeline name and before the rest
-        })
-    }
+    // Pipeline page version number dropdown
+    $('#version_select').on('change', function(){
+        document.location.href = $(this).val();
```

Comment:
Sorry, I am still more used to doing stuff in js than php 🙂

Reply:
No problem! Unsurprisingly the site is built using the web stack that I'm used to and have been building for years 👴🏻 

I think a lot of people would argue that we _should_ be doing this stuff in JS instead of PHP 😅